snapshot/lib/snapshot/runner.rb

```diff
@@ -9,6 +9,7 @@ class Runner
     # All the errors we experience while running snapshot
     attr_accessor :collected_errors
 
+    # rubocop:disable Metrics/AbcSize
```

Comment:
What do you think about disabling this check entirely? There are a number of places we disable it locally, and it doesn't seem to be changing the way we write our code.